core/src/main/scala/com/databricks/labs/remorph/discovery/queries.scala

```diff
@@ -5,7 +5,13 @@ import com.databricks.labs.remorph.intermediate.StructField
 import java.sql.Timestamp
 import java.time.Duration
 
-case class ExecutedQuery(id: String, timestamp: Timestamp, source: String, duration: Duration, user: String)
+case class QuerySpec(
```

Comment:
why do we need queryspec? don't change interfaces without a need

Reply:
how do we capture future metadata, today we add local file, tomorrow we might be getting other metadata trigger from ETL Job, etc, I feel we meed to have QuerySpec like outline or a Map[String,String] thoughts?

Reply:
move those fields to `ExecutedQuery` and we're good to merge

Reply:
Done